app/controllers/miq_ae_class_controller.rb

```diff
@@ -2084,7 +2084,8 @@ def fields_get_form_vars
         return
       end
       new_fields = {}
-      field_attributes.each_with_object({}) { |field| new_fields[field] = @edit[:new_field][field] }
+      field_attributes.each_with_object({}) { |field| new_fields[field] =
+        @edit[:new_field][field] || @edit[:new_field][field.to_sym]}
```

Comment:
@h-kataria  I have to check for data in this kind of hash `{:substitute=>true, "name"=>"Positive", :priority=>27, :class_id=>10000000000308, "aetype"=>"relationship", "datatype"=>"time"}`.